scripts/view_splitter.js

```diff
@@ -35,7 +32,8 @@ var viewSplitter = function(container, storageKey) {
     });
 
     function setSplitControls() {
-        splitImage.src = splitVertical ? proofIntData.buttonImages.imgHSplit : proofIntData.buttonImages.imgVSplit;
+        splitButton.style.backgroundImage = splitVertical ? `url("https://api.iconify.design/octicon:rows-24.svg")` : `url("https://api.iconify.design/octicon:columns-24.svg")`;
```

Comment:
can we avoid the 3rd party request? Our users are sensitive to these requests (why we got rid of momentjs)

Reply:
Yes, we should create a `pinc/3rdparty/iconify` directory with these two files if we want to use them to avoid the 3rd party API request. They are both licensed with the MIT license so we should include that in the directory and a `README.md` that calls out where we got them and that both files (by name) are covered by the MIT license. That lets us add more later with different (but still acceptable) licenses too.

Oh, and a `details.json` so we can give proper attribution.

Reply:
Done this.